versions.txt

```diff
@@ -1,6 +1,6 @@
 # Format:
 # module:released-version:current-version
 
-google-cloud-profiler:0.0.2:0.0.3-SNAPSHOT
-proto-google-cloud-profiler-v2:0.0.2:0.0.3-SNAPSHOT
-grpc-google-cloud-profiler-v2:0.0.2:0.0.3-SNAPSHOT
\ No newline at end of file
+google-cloud-profiler:0.1.2:0.1.3-SNAPSHOT
```

Comment:
These need to be what they should currently be, not the next version's values - release-please will propose the next version bumps